lib/Command/UpdateAccount.php

```diff
@@ -0,0 +1,164 @@
+
+                $smtpHost = $input->getOption(self::ARGUMENT_SMTP_HOST);
+                $smtpPort = $input->getOption(self::ARGUMENT_SMTP_PORT);
+                $smtpSslMode = $input->getOption(self::ARGUMENT_SMTP_SSL_MODE);
+                $smtpUser = $input->getOption(self::ARGUMENT_SMTP_USER);
+                $smtpPassword = $input->getOption(self::ARGUMENT_SMTP_PASSWORD);
+
+                $accounts = $this->mapper->findByUserId($userId);
+
+		foreach ($accounts as $account) {
+                # User may have have more than one email account. Only update the given one
+                        if ($account->getEmail() == $email) {
+                            $mailAccount = $account;
+                            break;
+                        }
+                  }
+                  if ($mailAccount) {
+                    //INBOUND
+                    if ($input->getOption(self::ARGUMENT_IMAP_HOST)) {
+                        $mailAccount->setInboundHost($imapHost);
+                    }
+
+                    if ($input->getOption(self::ARGUMENT_IMAP_PORT)) {
+                        $mailAccount->setInboundPort((int) $imapPort);
+                    }
+
+                    if ($input->getOption(self::ARGUMENT_IMAP_SSL_MODE)) {
+                      $mailAccount->setInboundSslMode($imapSslMode);
+                    }
+                    if ($input->getOption(self::ARGUMENT_IMAP_PASSWORD)) {
+                        $mailAccount->setInboundPassword($this->crypto->encrypt($imapPassword));
+                    }
+
+                    if ($input->getOption(self::ARGUMENT_SMTP_USER)) {
+                        $mailAccount->setInboundUser($imapUser);
+                    }
+
+                    // OUTBOUND
+  
+                    if ($input->getOption(self::ARGUMENT_SMTP_HOST)) {
+                        $mailAccount->setOutboundHost($smtpHost);
+                    }
+
+                    if ($input->getOption(self::ARGUMENT_SMTP_PORT)) {
+                        $mailAccount->setOutboundPort((int) $smtpPort);
+                    }
+                    if ($input->getOption(self::ARGUMENT_SMTP_SSL_MODE)) {
+                        $mailAccount->setOutboundSslMode($smtpSslMode);
+                    }
+                    
+                    if ($input->getOption(self::ARGUMENT_SMTP_PASSWORD)) {    
+                        $mailAccount->setOutboundPassword($this->crypto->encrypt($smtpPassword));
+                    }
+                    if ($input->getOption(self::ARGUMENT_SMTP_USER)) {
+                      $mailAccount->setOutboundUser($smtpUser);
+                    }
+
+                    $this->mapper->save($mailAccount);
+
+                    $output->writeln("<info>Account $email for user $userId succesfully updated </info>");
+
+                  } else {
+                        $output->writeln("<info>No Email Account $email found for user $userId </info>");
```

Comment:
```suggestion
                        $output->writeln("<info>No Email Account $email not found for user $userId </info>");
```

Reply:
The sentence is :  No mail account found....
So I did not add the  'not' you suggested.
If you think it would be more clear we could switch to:
'Mail account  not found'

Reply:
ha, sorry, I did not see the "No"! My apologies, you're fine!